packages/vue/src/components/molecules/SfComponentSelect/SfComponentSelect.stories.js

```diff
@@ -242,6 +242,19 @@ export default {
       defaultValue: "",
       description: "The content of the option",
     },
+    cancelLabel: {
```

Comment:
Can you add this prop also to the story's templates to make it work on Storybook?

Reply:
@AdamPawlinski was thinking about this too, but closing the dropdown seems like a native behavior for this component, so lets get back this in the next major version

Reply:
OK sounds good.